test/unit/utils.js

```diff
@@ -0,0 +1,34 @@
+import { fail } from 'assert';
+import chai from 'chai';
+import {
+  getBaseURL,
+  getTokenExpiresAtDate,
+  isWindowOrWorker,
+} from '../../src/utils.js';
+
+describe('Dropbox utils', () => {
+  describe('getBaseUrl', () => {
+    it('correctly sets base url when provided a host', () => {
+      const host = 'test';
+      const testUrl = getBaseURL(host);
+      chai.assert.equal(testUrl, 'https://test.dropboxapi.com/2/');
+    });
+  });
+
+  describe('getTokenExpiresAtDate', () => {
+    it('correctly sets when the token will expire', () => {
+      const expirationTime = 600;
+      const currentDate = new Date(Date.now());
+      const expiresDate = getTokenExpiresAtDate(expirationTime);
+      // adding a buffer of 100 to expiration time
```

Comment:
nit: 100 what? Since this formula isn't super obvious, just a quick one liner can help for posterity